watch-parties/src/main/webapp/chatscript.js

```diff
@@ -1,21 +1,39 @@
-//Timer under construction
-//document.getElementById("chatlog").onload = function() {setInterval(window.alert("hi"), 5000)}; 
-document.getElementById("sendButton").onclick = function() {addNewComment()}; 
+document.getElementById("sendButton").addEventListener("click", addNewComment);
 var formattedComment = "";
+var timer = window.setInterval(loadComments, 1000); //check comment updates every second
 
+// retrieves comments from the backend, loads using AJAX
+function loadComments() {
+  var xhttp;
+  xhttp = new XMLHttpRequest();
+  xhttp.onreadystatechange = function() {
+    if (this.readyState == 4 && this.status == 200) {
```

Comment:
Could make a constant for the ready state value with a name that tells us what the state is.

Also you can use https://docs.oracle.com/javaee/7/api/javax/ws/rs/core/Response.Status.html#OK for 200.

Reply:
This is a JS file so we can't use that particular constant.